.github/workflows/ci.yml

```diff
@@ -63,12 +56,12 @@ jobs:
     if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/metapep') }}
     runs-on: ubuntu-latest
     env:
-      NXF_VER: '20.04.0'
+      NXF_VER: '21.10.03'
       NXF_ANSI_LOG: false
     strategy:
       fail-fast: false
       matrix:
-        tests: ['test_mouse', 'test_assembly_only']   # add further test profiles here, will be run in parallel (but only with one nextflow version)
+        tests: ['test_assembly_only']   # add further test profiles here, will be run in parallel (but only with one nextflow version)
```

Comment:
All test profiles except `test` and `test_full` are listed here typically.

Reply:
Adding test_bins, other tests don't run at the moment because they need an ncbi_email and ncbi_key

Reply:
Ok